### PR TITLE
[ACS-3781] 878751 function cannot be performed by keyboard alone hiding the user groups

### PR DESCRIPTION
**Pull request by AleksanderSklorz**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/ACS-3781

**What is the new behaviour?**
User is able now to close window with users/groups using ESC key.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by MichalKinas — APPROVED** [review]

**Review by nikita-web-ua — APPROVED** [review]

File: lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.html [hunk 1]
```diff
@@ -42,6 +42,7 @@ <h3 class="adf-inherit-container-header">
                 data-automation-id="permission-info-button"
                 [adf-pop-over]="inheritedPermission"
                 [target]="target"
+                autofocusedElementSelector=".adf-sortable"
```
Comment (DenysVuika) [inline lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.html:45]: I suggest using square brackets to align with other input properties in the template
```suggestion
                [autofocusedElementSelector]="'.adf-sortable'"
```

File: lib/content-services/src/lib/permission-manager/components/pop-over.directive.ts [hunk 2]
```diff
@@ -34,6 +44,7 @@ export class PopOverDirective implements OnInit, OnDestroy, AfterViewInit {
     @Input('adf-pop-over') popOver!: TemplateRef<any>;
     @Input() target!: HTMLElement;
     @Input() panelClass = 'adf-permission-pop-over';
+    @Input() autofocusedElementSelector: string;
```
Comment (DenysVuika) [inline lib/content-services/src/lib/permission-manager/components/pop-over.directive.ts:47]: missing tests for new functionality

**Review by DenysVuika — CHANGES_REQUESTED** [review]

**Review by DenysVuika — APPROVED** [review]

File: lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.html (hunk 1 above)
Comment (AleksanderSklorz) [reply lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.html]: done

File: lib/content-services/src/lib/permission-manager/components/pop-over.directive.ts (hunk 2 above)
Comment (AleksanderSklorz) [reply lib/content-services/src/lib/permission-manager/components/pop-over.directive.ts:47]: done

**Pull request merged**